src/legacy/components/Receive.tsx

```diff
@@ -1,369 +0,0 @@
-/* eslint-disable */
```

Comment:
I don't think this file should be removed. Could you provide a reason?

Reply:
@br4in3x The PR with receive in the past, didnt remove this file. I checked and there's no reference to this file. His PR accidentally re-introduced `Receive.css.module`, when I removed that file, I noticed this and also removed this as well.